api/src/modules/Session/model.ts

```diff
@@ -1,57 +1,48 @@
 import type { PrismaClient } from '@prisma/client';
-import { inputObjectType, objectType } from 'nexus/dist';
 import { GraphQLError } from 'graphql';
 import ms from 'ms';
-import { token as tokenUtils } from '../../helpers';
+import { verifyToken, validateOriginAgainstAllowed, generateTokenAndSession } from './helpers';
 import { JWT_EXPIRATION_PERIOD } from '../../env';
-import { validateOriginAgainstAllowed, generateTokenAndSession } from './helpers';
 
-export const Session = objectType({
-  name: 'Session',
-  definition(t) {
-    t.nonNull.string('id');
-    t.nonNull.date('createdAt');
-    t.nonNull.string('createdBy');
-    t.date('referenceExpiryDate');
-    t.nonNull.string('referenceTokenId');
-    t.nonNull.boolean('isUserCreated');
-    t.string('name');
-    t.date('revokedAt');
-    t.string('allowedOrigins');
-  },
-});
+export function getSessionCrud(
+  prisma: Omit<PrismaClient, '$connect' | '$disconnect' | '$on' | '$transaction' | '$use'>,
```

Comment:
If this one type is used multiple times (>1) - should it be extracted and defined separately?

Reply:
Found suitable type inside prisma namespace